examples/policies/l4/l4_port_range.yaml

```
@@ -0,0 +1,14 @@
+apiVersion: "cilium.io/v2"
+kind: CiliumNetworkPolicy
+metadata:
+  name: "l4-port-range-rule"
+spec:
+  endpointSelector:
+    matchLabels:
+      app: myService
+  egress:
+    - toPorts:
+      - ports:
+        - port: "80"
+          endPort: 444
```

The API type described above is a string, shouldn't this be the following?

```suggestion
          endPort: "444"
```

---

(It's actually an int32 in the API)